pageserver/src/tenant/timeline.rs

```diff
@@ -2380,6 +2385,13 @@ impl Timeline {
             .unwrap_or(self.conf.default_tenant_conf.rel_size_v2_enabled)
     }
 
+    pub(crate) fn get_rel_size_v2_status(&self) -> Option<RelSizeMigration> {
```

Comment:
nit: would it be easier to just unwrap this to `Legacy` instead of propagating the `None`?